alert-policies/nrm4sap/bsHttpDest.yml

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
We can remove this line from all the alert policies.

Reply:
Modified alert policies accordingly.